### PR TITLE
Update with 

### PR DESCRIPTION
**Pull request by deepanjanroy**
@tdresser PTAL.

- I changed an FAQ question and added two more FAQs. 
- I didn't change the entire description of the algorithm, but called out the fact that the window is also quiet on the main thread. Is this enough or do you want the sliding window description?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ The precise algorithmic definition is as follows:
         *   In this case, reset the window start time to the end of this long task and continue growing the window.
     *   Or, last 5 seconds of the window is _network quiet_.
         *   Go to next step in this case.
-        *   Currently we call a time interval network quiet if it contains a maximum of two simultaneous network request at any given time. Forgiving up to two requests allows us to still reach network quiescence on sites that have long running network requests, for example to perform long polling.
+        *   Currently we call a time interval network quiet if it contains a maximum of two simultaneous resource requests at any given time. We exclude failed resource requests and resource requests that are not made with the GET HTTP method and Forgiving up to two requests allows us to still reach network quiescence on sites that have long running network requests, for example to perform long polling.
```
Comment (tdresser) [inline README.md]: Grammar gets funky around "Forgiving"

Comment (tdresser) [reply README.md]: Might be worth splitting this into two sentences?

**Review by tdresser — COMMENTED** [review]
Seems reasonable to me, but it might be worth getting @pmeenan's feedback.

**Comment by pmeenan** [comment]
LGTM.  Might be worth a note somewhere that the effect of resetting the idle at the point of a long task is that you end up with a window of at least 5 seconds of main-thread idle as well as network quiet.  That bit wasn't obvious to me when just reading through it.

File: README.md (hunk 1 above)
Comment (deepanjanroy) [reply README.md]: Done.

**Review by deepanjanroy — COMMENTED** [review]
Fixed grammar, and added note on effect of resetting idle at long task.

**Pull request merged**